### ISSUE TITLE
ALT text attribute missing from icon added via LO icon optional property

### ISSUE THREAD

**Issue by ronm123** [thexerteproject/xerteonlinetoolkits#892]
It was reported via the forums that a logo added via the icon optional property wasn't showing any alt text even when the description optional property has text added.

After investigating this I can see that in current develop xenith we have a check for icon alt text around line 1088 e.g.
`if (x_params.icTip != undefined && x_params.icTip != "") {
					$('#x_headerBlock img.x_icon').attr('alt', x_params.icTip);
				} else {
					$('#x_headerBlock img.x_icon').attr('aria-hidden', 'true');
				}`
However that if statement isn't being reached. I'm not sure how long that has been the case but I can see that with an icon and alt text added to the LO the logic reaches the if statement on line 1073 e.g. if (type==('LO') but it doesn't reach line 1088 where the check and setting of the alt text is located. I can get it working by moving that icTip check inside if (type==('LO') so around 1075 but there must be a reason for the later  $.ajax code where the check for icTip is currently located?

**Comment by torinfo** [comment]
@ronm123 I had a look, and it's something that I changed in a commit to fix offline functionality. 

I suppose that I meant to do the show and return if the type was 'LO' AND the package was an offline package, because than the ajax call is not sure to work. 

So I changed the code on line 1073 accordingly:

`if (type == 'LO' && xot_offline)`

It seems to work better now. Can you confirm?

**Comment by ronm123** [comment]
@torinfo I don't understand the logic. The check for and setting of the alt text should apply regardless of any other checks shouldn't it? Anyway I pulled down the change and the alt text now appears when viewing the project from Xerte and when viewing a project exported as archive but it isn't showing when the project is exported as offline. Unless I'm missing something?

**Comment by torinfo** [comment]
@ronm123 That is correct. It ill not work if the package is offline.

The first thing the rest of the code does is an ajax call, and we try to absolutley minimise calls to the internet for offline packages. I'll see if I can add the alt-text in the first if
